src/hooks/historikk/useDialogMedBehandlerHistorikk.ts

```diff
@@ -0,0 +1,59 @@
+import { HistorikkEvent } from "@/data/historikk/types/historikkTypes";
+import { useBehandlerdialogQuery } from "@/data/behandlerdialog/behandlerdialogQueryHooks";
+import {
+  MeldingDTO,
+  MeldingResponseDTO,
+} from "@/data/behandlerdialog/behandlerdialogTypes";
+import { meldingTypeTexts } from "@/data/behandlerdialog/behandlerdialogTexts";
+
+interface DialogMedBehandlerHistorikk {
+  isDialogMedBehandlerHistorikkLoading: boolean;
+  isDialogMedBehandlerHistorikkError: boolean;
+  dialogMedBehandlerHistorikk: HistorikkEvent[];
+}
+
+function avsenderText(dialogmelding: MeldingDTO): string {
+  return dialogmelding.innkommende
+    ? dialogmelding.behandlerNavn ?? "Mangler navn på behandler"
+    : dialogmelding.veilederIdent ?? "Mangler navn på veileder";
```

Comment:
Bør vel strengt tatt stå "Mangler ident på veileder" evt "Mangler veileder-ident" her

Reply:
Er for så vidt enig i det. Endrer til "Mangler ident på veileder"